src/app/pages/StoryPage/index.jsx

```diff
@@ -96,6 +142,9 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     byline: props => <StyledByline {...props} />,
     include: props => <Include {...props} />,
     social_embed: props => <SocialEmbed {...props} />,
+    wsoj: props => (
```

Comment:
Is there a reason this is `wsoj` and not `recommendations`?

Reply:
we're using what's been specified (as `wsoj`) [here in the preprocessor](https://github.com/bbc/simorgh/blob/latest/src/app/routes/cpsAsset/getInitialData/addRecommendationsBlock/index.js#L31)

Reply:
Thanks, Rowland. My concern is that we discussed moving away from the term `wsoj` and using `recommendations`, but if this has already been set then there's not much we can do about that in this PR. 😄